WordPress/gutenberg
# [Mobile] Add external link inline component

## Description
Add inline link to an external resource in the mobile app. It replaces the current `FooterMessageLink` Component since it does the same thing as the external link to create consistency with the Gutenberg components code base.

## How has this been tested?
1. Open the PR on a mobile device. Start a mobile dev environment. 
2. Open the editor and add the heading block. 
3. Open the block settings notice the new inline link. 

## Screenshots 
See Header Settings

<img width="404" alt="Screen Shot 2021-07-05 at 2 30 51 PM" src="https://user-images.githubusercontent.com/115071/124519690-e024a000-dd9e-11eb-9d28-92aa779cfca2.png">

Image block Alt Text settings.
<img width="395" alt="Screen Shot 2021-07-05 at 2 41 39 PM" src="https://user-images.githubusercontent.com/115071/124519770-1eba5a80-dd9f-11eb-8092-e208c7b0f82a.png">


## Types of changes
Enhancement. 

## Checklist:
